app/models/session.model.js

```diff
@@ -1,14 +1,14 @@
 module.exports = (sequelize, Sequelize) => {
-  const Session = sequelize.define("session", {
-    email: {
-      type: Sequelize.STRING,
-      allowNull: false,
-    },
-    expirationDate: {
-      type: Sequelize.DATE,
-      allowNull: false,
-    },
-  });
+    const Session = sequelize.define("session", {
+        email: {
```

Comment:
a `user` model object already has an `email` field and a `user` is associated with many `session` model objects, so why does `session` also have an `email` field?  it is expected to be different than the user's email?  if a user has many sessions will they each have the same or different emails?

Reply:
Since this was already in place and seems to be working with the recipe app, I definitely want to make this easy on ourselves and keep it in place. 
But to address you're question, the email in the session data is directly connected to the user logged in and is definitely redundant and doesn't really need to be there. Everything that I can see that concerns itself with the session doesn't care about the email in the session table. Since there is a foreign key to the user, the email is totally unnecessary. 
I only say keep it there for now because I haven't given this a thorough look to be 100% confident nothing would break by omitting the email from the session table. I would call this a "nice to have" tech debt cleanup item if we have time after all the app requirements are met.

Reply:
that's fair